ikasaneip/module/src/main/java/org/ikasan/module/service/ModuleInitialisationServiceImpl.java

```diff
@@ -495,6 +502,20 @@ private void createMetadata(Module<Flow> moduleRuntime, org.ikasan.topology.mode
         }
     }
 
+    private List<Component> distinctComponents(String flowName, Set<Component> components)
+    {
+        logger.info(String.format("Filtering distinct components for flow[%s]. Before:[%s].", flowName, components.size()));
```

Comment:
Use slf4j logger varargs rather than String.format

logger.info("Filtering distinct components for flow {}. Before: {}, flowName, components.size());

Reply:
Will do.